front/app/components/admin/NavigationTabs/Tab.tsx

```diff
@@ -72,22 +80,37 @@ type TabProps = {
   active: boolean;
   badge?: React.ReactNode;
   handleClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+  disabled?: boolean;
+  disabledTooltipText?: string;
 };
 
 const Tab = ({
   label,
   url,
-  active,
   badge,
   handleClick,
+  disabled,
+  disabledTooltipText,
   ...props
 }: TabProps) => (
-  <Container active={active} {...props}>
-    <Link to={url} onClick={handleClick}>
-      {label}
-      {badge && <>{badge}</>}
-    </Link>
-  </Container>
+  <Tippy
+    interactive={false}
+    placement="bottom"
+    theme={''}
+    disabled={!disabled}
+    maxWidth={350}
+    content={
+      <TooltipContentWrapper tippytheme="light">
+        {disabledTooltipText}
+      </TooltipContentWrapper>
+    }
+  >
+    <Container disable={disabled} {...props}>
```

Comment:
If the `disabled` prop name is used, the tooltip is not shown. Maybe related to the fact that it's a default HTML attribute https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled